app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -136,16 +132,11 @@ export const isAcuantCameraAccessFailure = (error: AcuantCaptureFailureError): e
  *
  */
 function getDocumentTypeLabel(documentType: AcuantDocumentType): AcuantDocumentTypeLabel | string {
-  switch (documentType) {
-    case 0:
-      return 'none';
-    case 1:
-      return 'id';
-    case 2:
-      return 'passport';
-    default:
-      return `An error in document type returned: ${documentType}`;
+  const label = AcuantDocumentType[documentType];
+  if (label === undefined) {
+    return `An error in document type returned: ${documentType}`;
   }
+  return label;
 }
```

Comment:
Could use a `??` operator to simplify this if we wanted.

```ts
const getDocumentTypeLabel = (documentType: AcuantDocumentType): AcuantDocumentTypeLabel | string =>
  AcuantDocumentType[documentType] ?? `An error in document type returned: ${documentType}`;
```

Reply:
great idea, updated in 1273a4e76226083b8cb1ef4d04de35939bf17f9e